src/providers/wfs/qgswfsshareddata.cpp

```diff
@@ -49,6 +49,8 @@ bool QgsWFSSharedData::isRestrictedToRequestBBOX() const
 QgsWFSSharedData *QgsWFSSharedData::clone() const
 {
   QgsWFSSharedData *copy = new QgsWFSSharedData( mURI.uri( true ) );
+  copy->mURI.setGetEndpoints(mURI.getGetEndpoints());
+  copy->mURI.setPostEndpoints(mURI.getPostEndpoints());	
```

Comment:
It seems to me that that below would be more robust. It might perhaps involve adding an assignment operator to QgsWFSDataSourceURI
```suggestion
  copy->mURI = mURI;
```